tests/tracing/instrumentation/test_action_executor.py

```diff
@@ -56,3 +60,30 @@ async def test_tracing_action_executor_run(
     assert captured_span.name == "MockActionExecutor.run"
 
     assert captured_span.attributes == expected
+
+
+@pytest.mark.asyncio
+async def test_instrument_action_executor_run_registers_tracer(
```

Comment:
Why is this marked as async? We're not using any async methods or functions in this test?

Reply:
I removed the async call and didn't update.

Reply:
Oh the action executor's `run` method is async, that's why this is required, i'd add a docstring or comment to describe this.

Reply:
yes, but i didn't call the method in the test.
I've updated the method to not be async.